source/include/actbl2.h

```diff
@@ -1418,6 +1419,21 @@ typedef struct acpi_mtmr_entry
 
 } ACPI_MTMR_ENTRY;
 
+/*******************************************************************************
+ *
+ * SDEI - Software Delegated Exception Interface Descriptor Table
+ *
+ * Conforms to "Software Delegated Exception Interface (SDEI)" ARM DEN0054A,
+ * May 8th, 2017. Copyright 2017 ARM Ltd.
```

Comment:
Hi Erik,

I picked the SLIC table as a template, it has a date and copyright statement like this. I guess Robin picked a different example.

Thanks,